components/doppler_marketing_automation/common/constants.mjs

```diff
@@ -0,0 +1,1009 @@
+export const GENDER_OPTIONS = [
+  {
+    label: "Female",
+    value: "F",
+  },
+  {
+    label: "Male",
+    value: "M",
+  },
+];
```

Comment:
Are these the only options for the prop `gender`? If so, I'd remove that prop from `add-update-subscriber` since it's optional, and there aren't enough options to be inclusive.